Dockerfile

```diff
@@ -63,7 +61,14 @@ ARG BUILD_OUTPUT_DIR
 WORKDIR /opt/taraxa/
 COPY conanfile.py .
 RUN conan remote add -f bincrafters "https://api.bintray.com/conan/bincrafters/public-conan"
-RUN conan install -if $BUILD_OUTPUT_DIR --build missing -s build_type=Debug .
+RUN conan profile new clang --detect
+RUN conan profile update settings.compiler=clang clang
+RUN conan profile update settings.compiler.version=10 clang
+RUN conan profile update settings.compiler.libcxx=libstdc++11 clang
+RUN conan profile update env.CC=/usr/lib/llvm-10/bin/clang clang
+RUN conan profile update env.CXX=/usr/lib/llvm-10/bin/clang++ clang
+
+RUN conan install -if $BUILD_OUTPUT_DIR --build missing -s build_type=Debug -pr=clang .
```

Comment:
Is there a reason for this not to be in a single `RUN` step?

Reply:
No. Changed